pkgs/top-level/darwin-packages.nix

```diff
@@ -108,9 +112,6 @@ impure-cmds // appleSourcePackages // chooseLibs // {
     stdenv = if stdenv.isDarwin then stdenv else pkgs.libcxxStdenv;
   };
 
-  # TODO: remove alias.
-  cf-private = self.apple_sdk.frameworks.CoreFoundation;
-
```

Comment:
Could you please do this in a separate PR? This could be merged much earlier than the SDK bump.

Reply:
See https://github.com/NixOS/nixpkgs/pull/229314